src/reactpy_django/templatetags/jinja.py

```diff
@@ -0,0 +1,67 @@
+# Copyright © 2023 Innovatie Ltd. All rights reserved.
+"""
+Jinja support.
+"""
+import typing as t
+
+from django.template import RequestContext, loader
+from jinja2 import pass_context
+from jinja2.ext import Extension
+from jinja2.runtime import Context, Undefined
+
+from .reactpy import component as djt_component
+from .. import config
+
+
+class ReactPyExtension(Extension):
+    """
+    Jinja has more expressive power than core Django's templates, and can
+    directly handle expansions such as:
+
+        {{ component(*args, **kwargs) }}
+    """
+    DJT_TEMPLATE = 'reactpy/component.html'
+    #
+    # Therefore, there is no new tag to parse().
+    #
+    tags = {}
+
+    def __init__(self, environment):
+        super().__init__(environment)
+        #
+        # All we need is to add global "component" to the environment.
+        #
+        environment.globals["component"] = self._jinja_component
+
+    @pass_context
+    def _jinja_component(self, __context: Context, dotted_path: str, *args: t.Any, host: str | None = None,
```

Comment:
May as well just accept `_jinja_component(self, __context: Context, *args, **kwargs)`.

That is, unless there's some tools out there for Jinja auto-complete within templates.

On that note, this method should be renamed to `def template_tag( ... )`.